packages/daimo-contract/src/shovel/config.json

```diff
@@ -6,6 +6,7 @@
       "name": "$CHAIN_NAME",
       "chain_id": "$CHAIN_ID",
       "url": "$CHAIN_RPC_URL",
+      "ws_url": "$CHAIN_RPC_WS_URL",
```

Comment:
proposal

```
"ws_url": "$CHAIN_RPC_URL_WS",
"urls": ["$CHAIN_RPC_URL", "$CHAIN_RPC_URL_BACKUP"],
```

^ i'll make the corresponding change in infra

Reply:
actually that second part is done in https://github.com/daimo-eth/daimo/pull/1035

shipping